tests/components/config/test_device_registry.py

```diff
@@ -160,6 +164,44 @@ async def test_update_device(hass, client, registry, payload_key, payload_value)
     assert isinstance(device.disabled_by, (helpers_dr.DeviceEntryDisabler, type(None)))
 
 
+async def test_update_aliases(hass, client, registry):
+    """Test update entry."""
+    device = registry.async_get_or_create(
+        config_entry_id="1234",
+        connections={("ethernet", "12:34:56:78:90:AB:CD:EF")},
+        identifiers={("bridgeid", "0123")},
+        manufacturer="manufacturer",
+        model="model",
+    )
+
+    assert not device.aliases == {}
+
+    aliases = ["alias_1", "alias_2"]
```

Comment:
Should we also test adding identical aliases ?
```python
aliases = ["alias", "alias"]
...
assert msg["result"]["aliases"] == aliases[0] # or ["alias"]
assert device.aliases == set(aliases)         # or set("alias")
```

Reply:
Good call, the test has been improved in 6a62dbc